packs/salt/actions/lib/utils.py

```diff
@@ -18,13 +21,48 @@
     "enabled": True,
     "entry_point": "runner.py"}
 
+local_action_meta = {
+    "name": "",
+    "parameters": {
+        "action": {
+            "type": "string",
+            "immutable": True,
+            "default": ""
+        },
+        "args": {
+            "type": "array",
+            "required": False
+        },
+        "kwargs": {
+            "type": "object",
+            "required": False
+        }
+    },
+    "runner_type": "run-python",
+    "description": "Run Salt Execution modules through Salt API",
+    "enabled": True,
+    "entry_point": "local.py"}
+
+
+def generate_actions():
```

Comment:
Would be nice to add a short section to the readme which describes how to use this script to re-generate the metadata files.

In addition to that, we should also add a warning to the automatically generated metadata files to make sure they are not edited manually (we can use YAML comment notation for that).